README.md

```diff
@@ -1,2 +1,157 @@
-# Hello-world
-I am creating this repository for HTML-CSS linters configuration.
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h3><b>Hello-World</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [Hello-World] <a name="about-project"></a>
+
+This project is about Microverse Linters configuration and ensuring that all linters checks pass, and also add & display a simple HTML text and basic CSS styling.
+
+**[Hello-World]** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+HTML5
+CSS3
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Version Control</summary>
+  <ul>
+    <li><a href="https://github.com/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Visual Studio Code</summary>
+  <ul>
+    <li><a href="https://code.visualstudio.com/">Express.js</a></li>
+  </ul>
+</details>
```

Comment:
- [ ] As suggested by the previous 🔙 reviewer you should only keep the sections relevant to your project. Please remove the following section from the Teck stack and update it according to your project’s tech stack e.g HTML and CSS. Also, Visual Studio and Version Control are not tech stack but you may add them into a separate section `Tools` 🔧 🪛